packages/xstate-solid/test/useMachine.test.tsx

```diff
@@ -54,7 +59,7 @@ describe('useMachine hook', () => {
             actions: assign({
               data: ({ event }) => event.output
             }),
-            guard: ({ event }) => event.output.length
+            guard: ({ event }) => !!event.output.length
```

Comment:
I noticed that the narrowed down event type is provided to `guard` here but it's not provided to `actions`. It's quite surprising since `TExpressionEvent` is used in both.